### PR TITLE
First commit to restful.example

### PR DESCRIPTION
**Pull request by ronsigal**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      <id>jboss-releases-repository</id>
+      <name>JBoss Releases Repository</name>
+      <url>https://repository.jboss.org/nexus/service/local/staging/deploy/maven2/</url>
+    </repository>
+    <snapshotRepository>
+      <id>jboss-snapshots-repository</id>
+      <name>JBoss Snapshots Repository</name>
+      <url>https://repository.jboss.org/nexus/content/repositories/snapshots/</url>
+    </snapshotRepository>
+  </distributionManagement>
```
Comment (jamezp) [inline pom.xml]: Is there a reason we need these?

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE web-app PUBLIC
```
Comment (jamezp) [inline src/main/webapp/WEB-INF/web.xml]: We shouldn't really need this file since all this is already done with annotations.

File: pom.xml (hunk 1 above)
Comment (ronsigal) [reply pom.xml]: Well, I don't know. At a RESTEasy meeting when you were away, I asked about deploying directly to JBoss nexus. I had copied stuff in the course of a release, but I don't remember doing it directly. Anyway, Jim mentioned this addition, and it worked, so it's there. I'm guessing setting the parent to org.jboss:jboss-parent would work?

File: src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (ronsigal) [reply src/main/webapp/WEB-INF/web.xml]: I think I used it to set the servlet name:
```
        <init-param>
            <param-name>javax.ws.rs.Application</param-name>
            <param-value>org.jboss.resteasy.example.ExampleApp</param-value>
        </init-param>
```
but I could do it differently.

File: pom.xml (hunk 1 above)
Comment (ronsigal) [reply pom.xml]: Ok, I set the parent to org.jboss:jboss-parent.

File: src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (ronsigal) [reply src/main/webapp/WEB-INF/web.xml]: Gone

File: pom.xml (hunk 1 above)
Comment (ronsigal) [reply pom.xml]: No, I need something in the pom.xml of  the generated project. I put back <distributionManagement>.

**Pull request merged**